state/protocol/seed/seed.go

```diff
@@ -1,44 +1,44 @@
 package seed
 
 import (
-	"encoding/binary"
 	"fmt"
 
 	"github.com/onflow/flow-go/consensus/hotstuff/packer"
+	"github.com/onflow/flow-go/crypto"
 	"github.com/onflow/flow-go/crypto/hash"
+	"github.com/onflow/flow-go/crypto/random"
 )
 
-// FromParentSignature reads the raw random seed from a main consensus QC sigData.
-// The sigData is an RLP encoded structure that is part of QuorumCertificate.
-// The indices can be used to generate task-specific seeds from the same signature.
-func FromParentSignature(indices []uint32, sigData []byte) ([]byte, error) {
-	// unpack sig data to extract random beacon sig
-	randomBeaconSig, err := packer.UnpackRandomBeaconSig(sigData)
+// PRGFromRandomSource returns a PRG seeded by the source of randomness of the protocol.
+// The customizer is used to generate a task-specific PRG (customizer in this implementation
+// is up to 12-bytes long).
+//
+// The function extracts first the source of randomness from sigData and then hashes
+// it to obtain a seed. Hashing is required to uniformize the entropy over the output,
+// since the source of randmoness is a cryptographic signature.
```

Comment:
Now the random source is passed into this function, not sure if this part of the comment is still relevant.